api/src/main/java/io/minio/MinioClient.java

```diff
@@ -2388,7 +2388,7 @@ public void copyObject(String bucketName, String objectName, Map<String,String>
       headerMap.putAll(copyConditions.getConditions());
     }
 
-    HttpResponse response = executePut(bucketName, objectName, headerMap, null, "", 0);
+    HttpResponse response = executePut(bucketName, objectName, headerMap, null, "", 0, false);
```

Comment:
Use overloaded `executePut()`/`executePost()` accepting `md5Required` argument to avoid unnecessary diffs.